packages/kit/src/core/adapt/test/index.js

```diff
@@ -91,7 +91,7 @@ suite('prerender', async () => {
 	};
 
 	/** @type {import('types/internal').BuildData} */
-	const build_data = { client: [], server: [], static: [], entries: ['/nested'] };
+	const build_data = { client: [], server: [], static: [], entries: ['/nested', '/noindex.html'] };
```

Comment:
`noindex.html` is kind of a confusing name in my opinion. I'd rather just call it something unrelated like `about.html`

Reply:
Ok, I'll change to `about.html`